test/cloudant.test.js

```diff
@@ -102,61 +102,73 @@ describe('cloudant connector', function() {
       Product.destroyAll(null, {limit: testUtil.QUERY_MAX}, done);
     });
 
-    it('updates a single instance with array props',
-      function(done) {
-        prod1.setAttribute('type', ['cinnamon raisin']);
-        prod1.setAttribute('releases', [4, 5, 6]);
-        prod1.setAttribute('foo', [{id: 3, name: 'bread3'}]);
-        Product.updateAll({id: 1}, prod1, function(err, res) {
-          if (err) return done(err);
-          Product.findById('1', function(err, res) {
+    it('updates a single instance with array props', function(done) {
+      prod1.setAttribute('type', ['cinnamon raisin']);
+      prod1.setAttribute('releases', [4, 5, 6]);
+      prod1.setAttribute('foo', [{id: 3, name: 'bread3'}]);
+      Product.updateAll({id: 1}, prod1, function(err, res) {
+        if (err) return done(err);
+        Product.findById('1', function(err, res) {
+          if (err) done(err);
+          res.name.should.equal(prod1.name);
+          res.price.should.equal(prod1.price);
+          res.releases.should.deepEqual([4, 5, 6]);
+          res.type.should.deepEqual(['cinnamon raisin']);
+          res.foo.should.deepEqual([{id: 3, name: 'bread3'}]);
+          Product.findById('2', function(err, res) {
```

Comment:
No idea :\ It was always there [testing both instances](https://github.com/strongloop/loopback-connector-cloudant/blob/master/test/cloudant.test.js#L154) :p